dev_requirements/requirements-tests.in

```diff
@@ -1,2 +1,3 @@
 pytest
 pytest-asyncio
+bo4e
```

Comment:
it's now a test-requirement only. not a package requirement anymore

Reply:
Hatte ich gesehen, daher ist auch die eine Model klasse aus der init geflog aber fürs Testen brauchte man die